app/controllers/assignment_invitations_controller.rb

```diff
@@ -1,4 +1,6 @@
 class AssignmentInvitationsController < InvitationsController
+  skip_before_action :set_organization, :authorize_organization_access
```

Comment:
This is actually leftover code from another idea I was spiking out. `set_organization` never actually happens in this controller.